pkgs/development/python-modules/fastmcp/default.nix

```diff
@@ -69,6 +69,11 @@ buildPythonPackage rec {
   disabledTests = [
     # AssertionError: assert 'INFO' == 'DEBUG'
     "test_temporary_settings"
+    # Permission denied: '/homeless-shelter'
+    "test_oauth_literal_sets_up_oauth_shttp"
+    "test_oauth_literal_pass_direct_to_transport"
+    "test_oauth_literal_sets_up_oauth_sse"
+    "test_oauth_literal_pass_direct_to_transport_sse"
```

Comment:
Here, the software tries to write (or read) in the `$HOME` directory.
However, in the nix sandbox, by default, `$HOME` is set to `/homeless-shelter`, a non-writable folder`.

To circumvent this, you can add `writableTmpDirAsHomeHook` to `nativeCheckInputs`.
This hook will export the `$HOME` variable to some temporary writable directory.